pyproject.toml

```diff
@@ -90,6 +90,7 @@ optional = [
   # optuna/visualization/param_importances.py.
   "scipy",  # optuna/samplers/_gp
   "torch; python_version<='3.12'", # TODO(gen740): Remove this line when 'torch', a dependency of 'optuna/_gp', supports Python 3.13
+  "grpcio",  # optuna/storages/grpc.
```

Comment:
We need to guarantee version consistency because I encountered an error caused by the version mismatch of `protobuf` between my environment and optuna's grpc implementation used (i.e., [5.28.1](https://github.com/optuna/optuna/blob/4fb4a6666ac96f8dfcf290f4be0aa00c489e4379/optuna/storages/grpc/_auto_generated/api_pb2.py#L5)).
So, `protobuf==5.28.1` should be added.